### ISSUE TITLE
[Accessibility]: Action items bar has lists with one or fewer items

### ISSUE THREAD

**Issue by tjkandala** [sourcegraph/sourcegraph#36855]
### Audit type

ARC Toolkit

### User journey audit issue

#34056

### Problem description

The ARC Toolkit identified that the action items bar is rendering list elements with only one list item.

![Screen Shot 2022-06-07 at 5 00 02 PM](https://user-images.githubusercontent.com/37420160/172707229-56b3c4bf-d945-4be9-b670-1be8df6c26e7.png)

### Expected behavior

Lists should have multiple items. Use elements other than `<ul>` as parents to single elements.

### Additional details

![Screen Shot 2022-06-08 at 4 05 03 PM](https://user-images.githubusercontent.com/37420160/172707012-eb944362-b7c1-40a6-b643-07180a374e65.png)

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @muratsu @jjinnii @ryankscott - the "team/integrations" label was applied to this issue.
